tests/templates/kuttl/listener/10-assert.yaml

```
@@ -46,13 +46,27 @@ spec:
 ---
 apiVersion: v1
 kind: Service
+metadata:
+  name: test-trino-coordinator-default-headless
+spec:
+  type: ClusterIP # cluster-internal - by trino op
```

There are no listener classes for the headless service. Same for metrics further down.

```suggestion
  type: ClusterIP
```